### ISSUE TITLE
Annoying warning

### ISSUE THREAD

**Issue by obazoud** [chef/ffi-yajl#46]
```
warn "ffi-yajl/json_gem is deprecated, these monkeypatches will be dropped shortly"
```

See coments : https://github.com/chef/ffi-yajl/commit/d41c232591806892ebcb78574cd81fd51b03e1d9

**Comment by lamont-granquist** [comment]
where is it showing up?  what version of chef/ohai/whatever?  and what code is calling require 'ffi-yajl/json_gem'?  there isn't any bug to fix here, an issue needs to be opened against the repos that are using this library.  "this is annoying" isn't a remotely actionable bug report.

**Comment by obazoud** [comment]
hi,

-where is it showing up?
when I use knife commands (knife node list for example)

-what version of chef/ohai/whatever?
I use chef 11.16.4

-what code is calling require 'ffi-yajl/json_gem'?
don't know

-there isn't any bug to fix here
as a end user, what am I supposed to do with this warning ?

-an issue needs to be opened against the repos that are using this library
well, the warn comes from ffi-yajl https://github.com/chef/ffi-yajl/blob/master/lib/ffi_yajl/json_gem.rb#L6

Why not change to debug ?

**Comment by lamont-granquist** [comment]
That was fixed in 11.18.0

https://github.com/chef/chef/commit/25930bc96580a94fc5ed84fd9d42b9ed9babebd4
